app/views/pets/index.html.erb

```diff
@@ -6,6 +6,10 @@
   <h4>Sex: <%= pet.sex %></h4>
   <h4>Shelter Name: <a href="/shelters/<%= pet.shelter_id %>"><%= pet.shelter_name %></a></h4>
   <%=button_to "Edit #{pet.name}'s info!", "/pets/#{pet.id}/edit", :method => :get%>
-  <p><%=button_to "Delete #{pet.name}", "/pets/#{pet.id}", :method => :delete%></p>
+  <% if pet.approved == false %>
+    <p><%=button_to "Delete #{pet.name}", "/pets/#{pet.id}", :method => :delete%></p>
+  <% elsif pet.approved == true %>
+    <h4>You cannot delete this pet while its application is approved.</h4>
+  <% end %>
```

Comment:
This conditional is added to remove the delete button if the pet has been approved.